src/Microsoft.Health.Fhir.Core/Features/Search/Expressions/RevIncludeExpression.cs

```diff
@@ -0,0 +1,46 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Features.Search.Expressions
+{
+    /// <summary>
+    /// Represents an include expression (where an additional resource is included based on a reference)
+    /// </summary>
+    public class RevIncludeExpression : IncludeBaseExpression
```

Comment:
For reverse chained expressions, we dont't subclass, but have a `public bool Reversed { get; }` property instead. What is the reason for doing it differently here?

Reply:
done (see new pr)